libraries/AP_Scheduler/PerfInfo.cpp

```diff
@@ -18,7 +18,14 @@ void AP::PerfInfo::reset()
     max_time = 0;
     min_time = 0;
     long_running = 0;
-    log_dropped = DataFlash_Class::instance()->num_dropped();
+
+    DataFlash_Class *dataflash_perf = DataFlash_Class::instance();
+    if (dataflash_perf == nullptr) {
+        log_dropped = 0xFFFFFFFF;
```

Comment:
0 would make more sense here IMO

Reply:
this value 0xFFFFFFFF has no sense and is illogical to reach indeed. 0 could reach in any case.